tests/entry/test_entry.py

```diff
@@ -17,12 +18,13 @@
     reason="Tests ANSYS_DPF_ACCEPT_LA",
 )
 def test_license_agr(restore_accept_la_env):
+    dpf.server.shutdown_global_server()
```

Comment:
@ansys-akarcher can you confirm that this not being here before, was the reason why we got a `KeyError` as a second exception, and now that you shutdown the global server first, you do get a `DPFSErverException` as a second exception even for older servers? Is this expected?